src/assets/scripts/client/measurement/MeasureTool.js

```diff
@@ -0,0 +1,363 @@
+            this._points[this._points.length - 1] = value;
+        } else {
+            this.addPoint(value);
+        }
+    }
+
+    // ------------------------------ PRIVATE ------------------------------
+
+    /**
+     * Gets the initial turn parameters that are needed to fly to the first fix
+     *
+     * @for MeasureTools
+     * @method _getInitialTurnParameters
+     * @returns {object}
+     * @private
+     */
+    _getInitialTurnParameters() {
+        const { aircraft } = this;
+
+        if (aircraft === null) {
+            return null;
+        }
+
+        const { groundSpeed, groundTrack } = aircraft;
+        const turnRate = 3; // Turn rate seems fixed at 3°/sec // groundSpeed > 250 ? 1.5 : 3;
+        const turnRadius = km(groundSpeed / (turnRate * 20 * Math.PI));
+        const start = aircraft.relativePosition;
+        const fix = this._getRelativePositionAtIndex(1);
+
+        // Get the turn direction by using the basic bearing to the next point
+        const bearingToFix = bearingToPoint(start, fix);
+        const isRHT = radians_normalize(bearingToFix - groundTrack) < Math.PI;
+        const direction = isRHT ? 1 : -1;
+
+        // The centre of the turn circle is offset to the left or right by 90°
+        const bearingToCenter = groundTrack + (direction * Math.PI / 2);
+        const center = [
+            start[0] + (turnRadius * Math.sin(bearingToCenter)),
+            start[1] + (turnRadius * Math.cos(bearingToCenter))
+        ];
+
+        // Turn exit (tangent from turn circle to the fix)
+        const centerToFixBearing = bearingToPoint(center, fix);
+        const centerToFixDistance = distance2d(center, fix);
+        const outboundCourse = centerToFixBearing + (direction * Math.asin(turnRadius / centerToFixDistance));
+
+        // Entry and exit angles
+        const entryAngle = bearingToCenter + Math.PI;
+        const exitAngle = outboundCourse - (direction * Math.PI / 2);
+
+        // The exit point is the point on the turn circle at the exit angle
+        const exitPoint = [
+            center[0] + (turnRadius * Math.sin(exitAngle)),
+            center[1] + (turnRadius * Math.cos(exitAngle))
+        ];
+
+        // Length around the arc
+        let turnAngle;
+        if (isRHT) {
+            turnAngle = exitAngle - entryAngle;
+        } else {
+            turnAngle = entryAngle - exitAngle;
+        }
+        const arcLength = turnRadius * radians_normalize(turnAngle);
+
+        return {
+            isRHT,
+            turnRadius,
+            arcLength,
+            entryAngle,
+            exitAngle,
+            center,
+            exitPoint
+        };
+    }
+
+    /**
+     * Get the text label that displays the distance and
+     * optional duration and bearings
+     *
+     * @param distance {number}
+     * @param duration {number}
+     * @param bearing {number|null}
+     * @private
+     */
+    _getLabel(distance, duration, bearing = null) {
```

Comment:
`_getLabel` -> `_buildLabel`

`get` has a very specific meaning and we aren't actually _*getting*_ anything here, we're calculating/building up.

Reply:
Same goes for `getPathInfo` and `_getInitialTurnParameters` then...